deploy/services/gitlab/docker-compose.yml

```diff
@@ -0,0 +1,30 @@
+# Reference: https://docs.gitlab.com/ee/install/docker/installation.html
+version: '3.6'
+services:
+  gitlab:
+    image: gitlab/gitlab-ce:17.3.1-ce.0
+    container_name: gitlab
+    restart: always
+    hostname: ${SERVER_DNS}
+    environment:
+      GITLAB_OMNIBUS_CONFIG: |
+        external_url '${URL_SCHEME}://${SERVER_DNS}/gitlab'
```

Comment:
perhaps `external_url` could be a variable in `.env`

Reply:
It looks like it's not possible to create compound variables in environment files. I was thinking the developer would then have to provide the site twice - in the `EXTERNAL_URL` variable and in `SERVER_DNS`